app/controllers/posts_controller.rb

```diff
@@ -17,14 +17,16 @@ def show
   def create
```

Comment:
Please solve the `N+1` problem when fetching all posts and their comments for a user in the `posts#index` action. You may learn more about the `N+1` problem [here](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0).